### DISCUSSION TITLE
How can I query RigidBodyPosition

### DISCUSSION THREAD

**Discussion by stevenliebregt** [jcornaz/heron#201] [Q&A] [answered]
Hi, so I'm looking if I should use heron or just plain rapier2d, and one of the things I came across was the following:

I want a camera that's following the player, and with rapier2d I could query the RigidBodyPositionComponent like this:
![image](https://user-images.githubusercontent.com/17197587/156124176-6f414a47-f96a-47b5-b14f-c2bf31711229.png)

But I can't do that with heron, or at least I haven't found out how yet, is this possible?

**Comment by jcornaz** [comment, marked as answer]
With heron, the rigid body position is the bevy transform. So you can simply query the `GlobalTransform` and access the translation.

**Reply by stevenliebregt** [reply]
Thanks!
